daemon/plugin_p.h

```diff
@@ -45,6 +45,7 @@ class PluginHelper : public QPluginLoader
 
     template <typename Plugin>
     Plugin* storeAs(QObject *obj, QMap<QString, Plugin*> *store,
+                    const QVariantMap &parameters,
```

Comment:
Why not keep a const reference as a private member of PluginHelper and associate it at construction time? So like that we avoid to pass the same parameter argument at each storeAs() call. After all, parameter argument is the same for a single PluginHelper instance whatever we try to load it as. I would have like to do this for the category also but was not fluent enough in C++ to understand what should be the type of the private member…

Reply:
Sounds reasonable. Let me see if we can do that.